lib/open_project/webhooks/engine.rb

```diff
@@ -27,7 +27,7 @@ class Engine < ::Rails::Engine
     config.before_configuration do |app|
       # This is required for the routes to be loaded first as the routes should
       # be prepended so they take precedence over the core.
-      app.config.paths['config/routes'].unshift File.join(File.dirname(__FILE__), "..", "..", "..", "config", "routes.rb")
+      app.config.paths['config/routes.rb'].unshift File.join(File.dirname(__FILE__), "..", "..", "..", "config", "routes.rb")
```

Comment:
Line is too long. [125/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.